snippets/nix.json

```
@@ -82,6 +82,86 @@
         ],
         "description": "Nixpkgs' stdenv.mkDerivation template"
     },
+    "pkgs.mkShell": {
+        "prefix": "mks",
+        "body": [
+            "pkgs.mkShell {",
+            "  shellHook = ''",
+            "    $1",
+            "  '';",
+            "",
+            "  env = {",
+            "    $2",
+            "  };",
+            "",
+            "  packages = [",
+            "    $3",
+            "  ];",
+            "}"
```

```suggestion
            "pkgs.mkShell {",
            "  packages = [",
            "    $1",
            "  ];",
            "}"
            "",
            "  env = {",
            "    $2",
            "  };",
            "",
            "  shellHook = ''",
            "    $3",
            "  '';",
```

Please reorganize it to this.

---

Because most users will want to touch packages first, env second and shell hook third.